### PR TITLE
fix: store in 13.32 #854

### PR DESCRIPTION
**Pull request by kokekanon**
# Description
https://github.com/mehah/otclient/issues/854

13.32
image
13.40
image

### **Actual**

Do this and that doesn't happens

### **Expected**

Do this and that happens

## Fixes

\# (issue)

## Type of change

Please delete options that are not relevant.

  - [ ] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

  - [ ] Test A
  - [ ] Test B

**Test Configuration**:

  - Server Version:
  - Client:
  - Operating System:

## Checklist

  - [ ] My code follows the style guidelines of this project
  - [ ] I have performed a self-review of my own code
  - [ ] I checked the PR checks reports
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [ ] My changes generate no new warnings
  - [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by kokekanon** [comment]
ups sorry canary 2847

File: modules/game_mainpanel/mainpanel.lua [hunk 1]
```diff
@@ -163,7 +163,7 @@ function optionsController:onInit()
 end
 
 function toggleStore()
-    if g_game.getFeature(GamePurseSlot) then
+    if g_game.getFeature(GamePurseSlot) and g_game.getClientVersion() >= 1332 then
```
Comment (luanluciano93) [inline modules/game_mainpanel/mainpanel.lua]: is this correct? Do you make the store work only for versions 13.32 or above?

**Review by luanluciano93 — CHANGES_REQUESTED** [review]

File: modules/game_mainpanel/mainpanel.lua (hunk 1 above)
Comment (kokekanon) [reply modules/game_mainpanel/mainpanel.lua]: store was tested at 13.32 and 13.40
as far as I know tfs (13.10) doesn't have a store.

a guy talked to me on discord he had 10.98(have gamepurseslot). (But 10.98 doesn't have a store. test in blacktek).


That if is for:
in that case user with 10.98 open the game shop from v8. Not store


but if you talk about canary versions like 13.21 , 13.16 or something similar. yes that "if", is incorrect.
but I have not tested

Comment (luanluciano93) [reply modules/game_mainpanel/mainpanel.lua]: the GamePurseSlot feature is added in version >= 953, that is, its check is: "if version >= 953 AND version >= 1332 ... "

**Review by luanluciano93 — APPROVED** [review]

**Pull request merged**